src/app/components/attestation/attestations-view.jsx

```diff
@@ -109,68 +109,70 @@ function ChplAttestationsView({ developer: initialDeveloper, dispatch }) {
-                          && (
-                            <TableCell>
-                              <span className="sr-only">View Details</span>
-                            </TableCell>
-                          )}
-                      </TableRow>
-                    </TableHead>
-                    <TableBody>
-                      { attestations
-                        .filter((att) => att.status === 'ATTESTATIONS_SUBMITTED' || canSeeUnsubmittedAttestationData())
-                        .map((item) => (
-                          <TableRow key={item.id ?? item.attestationPeriod.id}>
-                            <TableCell>
-                              { getDisplayDateFormat(item.attestationPeriod.periodStart) }
-                              {' '}
-                              to
-                              {' '}
-                              { getDisplayDateFormat(item.attestationPeriod.periodEnd) }
-                            </TableCell>
-                            <TableCell>
-                              { item.statusText }
-                            </TableCell>
-                            { canSeeAttestationData()
-                              && (
-                                <TableCell>
-                                  { item.status === 'ATTESTATIONS_SUBMITTED'
-                                    ? (
-                                      <IconButton
-                                        color="primary"
-                                        variant="contained"
-                                        onClick={() => viewAttestations(item)}
-                                        aria-label={`View attestations for period ending ${item.attestationPeriod.periodEnd}`}
-                                      >
-                                        <VisibilityIcon color="primary" />
-                                      </IconButton>
-                                    ) : (
-                                      <IconButton
-                                        color="primary"
-                                        variant="contained"
-                                        onClick={() => setExceptionPeriod(item.attestationPeriod)}
-                                        aria-label={`Create attestations exception for period ending ${item.attestationPeriod.periodEnd}`}
-                                        disabled={!canCreateException}
-                                      >
-                                        <AddIcon color="primary" />
-                                      </IconButton>
-                                    )}
-                                </TableCell>
-                              )}
-                          </TableRow>
-                        ))}
-                    </TableBody>
-                  </Table>
-                </TableContainer>
+                <Card>
+                  <TableContainer component={Paper}>
```

Comment:
Same question as the attestation-view file, but also, why does this one have a `Paper` component instead of a Card?